services/vehicle-assembly-process-defect-detection-model-service/app/core/dependencies.py

```
@@ -0,0 +1,375 @@
+                "message": "모델 매니저가 초기화되지 않았습니다. 서비스를 재시작해주세요.",
+                "error_code": "MODEL_MANAGER_NOT_INITIALIZED",
+                "timestamp": datetime.now().isoformat()
+            }
+        )
+
+    try:
+        is_loaded = _model_manager.is_loaded
+        logger.info(f"모델 로드 상태: {is_loaded}")
+    except Exception as e:
+        logger.error(f"is_loaded 접근 중 오류: {e}")
+        raise HTTPException(...)
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)